api/namespace/application/application_ns.py

```diff
@@ -1,10 +1,12 @@
-from flask_restx import fields
 from flask_restx import Namespace
 
-
 """
 Namespace acts as part of an api (with the same methods as api)
 """
 
 
-search_ns = Namespace("search", description="application search operations")
\ No newline at end of file
+application_ns = Namespace("application", path="", description="single application operations")
```

Comment:
Nice that there are separate endpoints for single and bulk operations.

Reply:
agreed - much nicer to call. great work Harry